File: 1791-richest-customer-wealth/1791-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int ans = 0 ;
        for (int i = 0; i < accounts.size(); ++i) {
            int cur = 0;
            for (int j = 0; j < accounts[0].size(); ++j) {
                cur += accounts[i][j];
            }
            ans = max(ans, cur);
        }
        return ans;

    }
};

File: 1791-richest-customer-wealth/1791-richest-customer-wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
  let ans = 0
  for (let account of accounts) {
    let cur = 0
    for (let num of account) {
      cur += num
    }
    ans = Math.max(ans, cur)
  }
  return ans
};